pkg/http/http.go

```diff
@@ -13,8 +12,8 @@ import (
 type Getter interface {
 	GetAll(fileURL string) ([]byte, error)
 	GetAllWithContext(ctx context.Context, fileURL string) ([]byte, error)
-	GetWithChecksum(fileURL string) (ReadCloserWithChecksum, error)
-	GetWithChecksumAndContext(ctx context.Context, fileURL string) (ReadCloserWithChecksum, error)
+	GetWithChecksum(fileURL string, ChecksumHasher hash.Hash) (ReadCloserWithChecksum, error)
```

Comment:
Not sure an interface needs to give names to args at all, but if then I'd prefer it starting with a lowercase letter.

```suggestion
	GetWithChecksum(fileURL string, checksumHasher hash.Hash) (ReadCloserWithChecksum, error)
```

Reply:
Done.